src/main/java/frc/robot/commands/intake/AutoIntake.java

```diff
@@ -0,0 +1,53 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands.intake;
+
+import frc.robot.commands.intake.ManualIntake;
+import frc.robot.subsystems.Intake;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Settings.Constants;
+
+public class AutoIntake extends CommandBase {
+  /**
+   * Creates a new AutoIntake.
+   */
+  Intake m_Intake;
+
+  public AutoIntake(Intake intake) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_Intake = intake;
+    addRequirements(intake);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_Intake.StartIntakeMotor(Constants.Intake.autoIntakeSpeed);
+    m_Intake.toggleIntakeSolenoid();
```

Comment:
Many issues,

1. It's going to toggle every 20ms, therefore, flip flip flip. Do this in initialize. Also, you have to check that the intake is in before you flip out.